lambdas/handlers/search_patient_details_handler.py

```diff
@@ -1,15 +1,22 @@
 import logging
+import os
 from json import JSONDecodeError
 
 from pydantic import ValidationError
+from requests import HTTPError
 from services.pds_api_service import PdsApiService
 from utils.exceptions import (InvalidResourceIdException,
                               PatientNotFoundException, PdsErrorException)
 from utils.lambda_response import ApiGatewayResponse
 
+from services.ssm_service import SSMService
+
+from services.mock_pds_service import MockPdsApiService
+
 logger = logging.getLogger()
 logger.setLevel(logging.INFO)
-
+def get_pds_service():
+    return PdsApiService if not bool(os.getenv("PDS_FHIR_IS_STUBBED")) else MockPdsApiService
 
 def lambda_handler(event, context):
```

Comment:
Maybe worth updating this to match the other lambdas by using the decorators for validation, then we can remove the need to validate nhs number in the mock pds service

Reply:
👍 to this - just add the following above the method declaration:

`@validate_patient_id`

Reply:
Done